lib/node_modules/@stdlib/stats/base/svarianceyc/examples/index.js

```diff
@@ -18,18 +18,14 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float32Array = require( '@stdlib/array/float32' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
 var svarianceyc = require( './../lib' );
 
 var x;
-var i;
 
-x = new Float32Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-	x[ i ] = round( (randu()*100.0) - 50.0 );
-}
+x = discreteUniform( 10, -50, 50, {
+	'dtype': 'float32'
+});
```

Comment:
```suggestion
var x = discreteUniform( 10, -50, 50, {
	'dtype': 'float32'
});
```
kindly make this change in the `README.md` examples too if applicable